config/cesm/config_grids.xml

```diff
@@ -179,18 +172,25 @@
       <mask>gx1v7</mask>
     </model_grid>
 
-    <model_grid alias="T42_T42_mg16" not_compset="_POP">
-      <grid name="atm">T42</grid>
-      <grid name="lnd">T42</grid>
-      <grid name="ocnice">T42</grid>
+    <model_grid alias="T85_T85_mg16" not_compset="_POP">
+      <grid name="atm">T85</grid>
+      <grid name="lnd">T85</grid>
+      <grid name="ocnice">T85</grid>
       <mask>gx1v6</mask>
     </model_grid>
 
-    <model_grid alias="T85_T85" not_compset="_POP">
+    <model_grid alias="T85_T85_mg17" not_compset="_POP">
       <grid name="atm">T85</grid>
       <grid name="lnd">T85</grid>
       <grid name="ocnice">T85</grid>
-      <mask>usgs</mask>
+      <mask>gx1v7</mask>
+    </model_grid>
+
+    <model_grid alias="T42_T42_mg16" not_compset="_POP">
+      <grid name="atm">T42</grid>
+      <grid name="lnd">T42</grid>
+      <grid name="ocnice">T42</grid>
+      <mask>gx1v6</mask>
```

Comment:
Is there a reason this is down here instead of with the other T42_T42 grids? When working on this file, it is very helpful (to me at least) to have these in some semblance of order.

Reply:
SCAM does need T42 (we are currently using that in our tests), so it needs to remain

Reply:
SCAM does need T42 (we are currently using that in our tests), so it needs to remain